<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gammaray::Internal::GammarayConfigWidget</class>
 <widget class="QWidget" name="Gammaray::Internal::GammarayConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>312</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="gammarayOptions">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Generic Settings</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="injectorLabel">
        <property name="text">
         <string>Injector type:</string>
        </property>
        <property name="buddy">
         <cstring>injectorCombo</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="injectorCombo"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>GammaRay versions</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTreeWidget" name="qtVersionTable">
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <attribute name="headerStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Qt Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>qmake Location</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>gammaray Location</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="Utils::DetailsWidget" name="qtInfoWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::DetailsWidget</class>
   <extends>QWidget</extends>
   <header location="global">utils/detailswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
